docs/getting-started/configuring-your-repository.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Configuring your repository
+---
+
+# Configuring your repository
+
+Once you've [added your first repository](codacy-quickstart.md), **it's important that you configure it** to best match your use case and preferences. An optimal configuration integrates code analysis seamlessly into your development pipeline while avoiding false positives. This encourages your team to follow best practices.
```

Comment:
It ended up rather short. I hope all the elements are there, especially assertiveness and value to the user.

Reply:
This is going in the right direction! :slightly_smiling_face: 

Here are some ideas to further improve the intro:

- I don't think we need the bold to reinforce saying "it's important that you configure your repository", after all, we have an entire page written about it. Instead, it's better to present the advantages of doing these configurations.
- Regarding "integrates code analysis seamlessly into your development pipeline", this will actually only be achieved on the last page "Integrating Codacy with your Git provider".

Here's a suggestion for applying this feedback:

```suggestion
Once you've [added your first repository](codacy-quickstart.md), it's important that you configure the analysis tools to match the use cases of your team and avoid any distracting false positives. In particular, you should configure any coding conventions and best practices that your team may already be following or that you want to enforce.
```